front/src/modules/ui/navbar/components/NavItem.tsx

```diff
@@ -134,7 +134,7 @@ function NavItem({
       danger={danger}
       soon={soon}
     >
-      {Icon && <Icon size={theme.icon.size.md} />}
```

Comment:
Please change the IconComponent type to add a `stroke: number` property.

We need to pass `<Icon size={theme.icon.size.md} stroke={theme.icon.stroke.sm />`

Reply:
@lucasbordeau pushed the changes [41826fb](https://github.com/twentyhq/twenty/pull/1560/commits/41826fb8b44e0f68ab9565e84062304bbcea0e9b)